6/run-client-docker.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+port=$1
+ADVERTISEDIP=`ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v -m1 '127.0.0.1'`
+GATEWAYPORT=3001
+
+docker build -t client-cluster -f ./ops/Api/Dockerfile ./ &&
+  docker run -d -p $port:80 -e ADVERTISEDIP=$ADVERTISEDIP  -e GATEWAYPORT=$GATEWAYPORT --rm client-cluster
```

Comment:
This client is perfectly fine to run in the background, unlike console clients from previous examples.

Reply:
I thought the port is `5000`?

Reply:
from docker we expose port 80 and map it to port 5000. Thats correct. Updated that script now, port was a script arg.